docs/map-making/upgrading-1-8-maps.md

```diff
@@ -18,158 +26,128 @@ Within every game (XML) file, inside the "games" folder inside the main folder o
 
 ### Changes of not already deprecated game (XML) file items:
 
-_(these are the main changes on game file items that were not deprecated, so you normally need to change these only, within the game file)_
+_(these are the main changes on game file items that were not deprecated, so you normally need to change
+these only, within the game file)_
 
 Within every game (XML) file, inside the "games" folder inside the main folder of the map, change:
 
-`attatchment`:`attachment`
-
-`Attatchment`:`Attachment`
-
-`attatchTo=`:`attachTo=`
-
-`="isImpassible"`:`="isImpassable"`
-
-`="conditionType" value="XOR"`:`="conditionType" value="1"`
-
-`="turns"`:`="rounds"`
-
-`="Battleships repair at end of round"`:`="Units Repair Hits End Turn"`
-
-`="Battleships repair at beginning of round"`:`="Units Repair Hits Start Turn"`
-
-`="Naval Bombard Casualties Return Fire Restricted"`:`="Naval Bombard Casualties Return Fire"`
-
-`="SBR Affects Unit Production"`:`="Damage From Bombing Done To Units Instead Of Territories"`
+- `attatchment` >> `attachment`
+- `Attatchment` >> `Attachment`
+- `attatchTo=` >> `attachTo=`
+- `="isImpassible"`>> `="isImpassable"`
```

Comment:
Here the format is missing spacing before the ">>" (compared to the rest).

Reply:
Quite the eagle eye :eagle:   :smile:  :+1: